npm/vite-dev-server/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { EventEmitter } from 'events'
+import { debug as debugFn } from 'debug'
+import { start as createDevServer } from './startServer'
+import { UserConfig } from 'vite'
+const debug = debugFn('cypress:vite-dev-server:vite')
+
+interface Options {
```

Comment:
We need to have some kind of shared API for all the servers that needs to be implemented. Probably it should include the wrapper around dev-server-events and this options